### PR TITLE
95 update all programs and summary json documents

### LINKED ISSUE #95: [FEATURE] Update all programs and summary JSON documents
**Is your feature request related to a problem? Please describe.**
Currently, the top-line numbers are not correct.  

**Describe the solution you'd like**
Consolidate individual CSV files, and update all programs and summary JSON document based on the current individual CSV files. This is to temporarily update the home page while the data in the database is getting ready for all programs.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sandeep-ps**

### CODE REVIEW

**Review by pengyin-shan — APPROVED** [review]
Approve this PR. It looks like the Crop Insurance numbers have some noticeable changes in `all programs` compared to the current version. I would suggest double-checking with Johnthan just as a confirmation. Everything looks good

File: all-programs-summary-new/allprograms.json [hunk 1]
```diff
@@ -0,0 +1,1718 @@
...
+    "Title I 2021": 0.0,
+    "Title I 2022": 0.0,
+    "Title I Total": 378836.0,
+    "Title II 2018": 7662394.39,
+    "Title II 2019": 6729781.45,
+    "Title II 2020": 8570223.77,
+    "Title II 2021": 9606129.47,
+    "Title II 2022": 8893772.0,
+    "Title II Total": 41462301.08,
+    "Crop Insurance 2018": 28332677.0,
```
Comment (navarroc) [inline all-programs-summary-new/allprograms.json]: Crop insurance doesn't look right. If I am reading this right, Alaska shows 28,332,677 for 2018, but our current map hover shows 81,000

Comment (pengyin-shan) [reply all-programs-summary-new/allprograms.json]: yes that is also what I was concerned about...did we change the calculation method of Crop Insurance @sandeep-ps ?

File: all-programs-summary-new/allprograms.json [hunk 2]
```diff
@@ -0,0 +1,1718 @@
...
+    "Title I 2021": 20372228.77,
+    "Title I 2022": 440010.11,
+    "Title I Total": 248932672.12,
+    "Title II 2018": 34646573.12,
+    "Title II 2019": 36603103.91,
+    "Title II 2020": 39102035.67,
+    "Title II 2021": 51295873.43,
+    "Title II 2022": 36045724.0,
+    "Title II Total": 197693310.13,
+    "Crop Insurance 2018": 81177.0,
```
Comment (navarroc) [inline all-programs-summary-new/allprograms.json]: It looks like AL has the data for Alaska for crop insurance. 81,177 look like what we show now on dev

File: all-programs-summary-new/allprograms.json (hunk 1 above)
Comment (sandeep-ps) [reply all-programs-summary-new/allprograms.json]: Thanks for catching this. I will look into this today. I'm guessing this is a mixup between AK and AL.

Comment (sandeep-ps) [reply all-programs-summary-new/allprograms.json]: Yes, there was a mix up when importing all program numbers into the All Programs (Final) sheet from Crop Insurance (Main) sheet. Only the data for crop insurance in those two states were affected. The grand totals doesn't change too. I have fixed the push in data-import PR and this PR. Please verify and resolve this conversation, if it looks good. Thanks for finding this.

**Comment by sandeep-ps** [comment]
@navarroc or @pengyin-shan, Could you please take another look at this PR? I have fixed the issue that you identified earlier. Thanks.

**Comment by pengyin-shan** [comment]
> @navarroc or @pengyin-shan, Could you please take another look at this PR? I have fixed the issue that you identified earlier. Thanks.

thanks for fixing @sandeep-ps , looks good to me now. We can ask Johnathan to confirm again after deploying to dev

**Pull request merged**